app/Models/Blessing.php

```diff
@@ -12,7 +12,6 @@ class Blessing extends Model
     use HasFactory;
 
     protected $fillable = [
-        'id',
```

Comment:
This is required since ids are not created by the DB since I am using snowflakes with Verbs

Reply:
Shouldn't be. It should be automatically filled by the model logic, vs fillable which is more answering "what can the user fill". Their source code examples seem to support this. It should be that you only need to cast it to the Snowflake class

```php
class Example extends Model
{
    // Auto-generate Snowflake for new models
    use HasSnowflakes;
    
    // Any attribute can be cast to a `Snowflake` (or `Sonyflake`)
    protected $casts = [
        'id' => Snowflake::class,
    ];
}

$example = Example::create();

$example->id instanceof Snowflake; // true

echo $example->id; // 65898467809951744
```